src/componets/Form/Form.tsx

```diff
@@ -0,0 +1,360 @@
+    alignItems: "center",
+  };
+
+  return (
+    <Box component="form" onSubmit={handleSubmit} sx={BoxFormStyle}>
+      <Typography
+        variant="h5"
+        sx={{ textAlign: "center", color: "primary.main" }}
+      >
+        Car adding form
+      </Typography>
+      <TextField
+        label="Brand"
+        name="brand"
+        value={formData.brand}
+        onChange={handleInputChange}
+        required
+        sx={TextFieldStyle}
+      />
+      <TextField
+        label="Model"
+        name="model"
+        value={formData.model}
+        onChange={handleInputChange}
+        required
+        sx={TextFieldStyle}
+      />
+      <TextField
+        label="Year"
+        name="year"
+        type="number"
+        value={formData.year}
+        onChange={handleInputChange}
+        required
+        InputProps={{ inputProps: { min: 2000, max: 2024 } }}
+        sx={TextFieldStyle}
+      />
+      <FormControl required sx={TextFieldStyle}>
+        <InputLabel>Body Type</InputLabel>
+        <Select
+          name="body_type"
+          value={formData.body_type}
+          onChange={handleSelectChange}
+          sx={{ width: "100%" }}
+        >
+          <MenuItem value="">
+            <em>Select...</em>
+          </MenuItem>
+          {bodyTypes.map((type) => (
+            <MenuItem key={type} value={type}>
+              {type}
+            </MenuItem>
+          ))}
+        </Select>
+      </FormControl>
+      <TextField
+        label="Mileage"
+        name="mileage_km"
+        type="number"
+        value={formData.mileage_km}
+        onChange={handleInputChange}
+        required
+        InputProps={{ inputProps: { min: 1, max: 1000 }, endAdornment: <InputAdornment position="end">1,000 km</InputAdornment> }}
+        sx={TextFieldStyle}
+        error={!!errors.mileage_km}
+        helperText={errors.mileage_km}
+/>
+      <FormControl component="fieldset" required sx={TextFieldStyle}>
+        <FormLabel component="legend">Gearbox</FormLabel>
+        <RadioGroup
+          name="gearbox"
+          value={formData.gearbox}
+          onChange={handleInputChange}
+          sx={RadioGroupStyle}
+        >
+          <RadioFormControlLabel
+            value="Manual"
+            label="Manual"
+            sx={RadioStyle}
+            required
+          />
+          <RadioFormControlLabel
+            value="Automatic"
+            label="Automatic"
+            sx={RadioStyle}
+            required
+          />
+        </RadioGroup>
+      </FormControl>
+      <FormControl component="fieldset" required sx={TextFieldStyle}>
+        <FormLabel component="legend">Fuel</FormLabel>
+        <RadioGroup
+          name="fuel"
+          value={formData.fuel}
+          onChange={handleInputChange}
+          sx={RadioGroupStyle}
+        >
+          <RadioFormControlLabel
+            value="Petrol"
+            label="Petrol"
+            sx={RadioStyle}
+            required
```

Comment:
you keep repeating the passing of styles to the component in all places it is used. Put these styles inside the component itself